tests/x11/yast2_lan_restart.pm

```diff
@@ -90,9 +90,9 @@ sub run {
     unless (is_network_manager_default) {
         # Run detailed check only if explicitly configured in the test suite
         check_etc_hosts_update() if get_var('VALIDATE_ETC_HOSTS');
-        verify_network_configuration(\&check_network_card_setup_tabs);
+        verify_network_configuration(\&check_network_card_setup_tabs, 'reload');
         verify_network_configuration(\&check_default_gateway);
-        verify_network_configuration(\&change_hw_device_name, 'dyn0', 'restart');
+        verify_network_configuration(\&change_hw_device_name, 'reload', 'dyn0');
```

Comment:
It's not that easy, SLE 15 SP1 has old behavior still, so we expect restart there.

Reply:
@dzedro i would like your input here as i cant find a job to verify this

Reply:
https://openqa.suse.de/tests/4100059 sle15sp1
https://openqa.suse.de/tests/4100060 sle15
https://openqa.suse.de/tests/4100061 sle12sp5

Reply:
@dzedro take another look whenever you have the time.

Reply:
pass https://openqa.suse.de/tests/4123051 15sp1
fail https://openqa.suse.de/tests/4123052 15
fail https://openqa.suse.de/tests/4123053 12sp5